### PR TITLE
replaced logo elements with updated version.

### PR DESCRIPTION
**Pull request by swoocn**
New logo elements as provided/ attached in Trello.

### CODE REVIEW

File: frontend/src/favicon.ico
Comment (zolcsi) [inline frontend/src/favicon.ico:1]: I think this is way too big for a favicon. As it turns out, the approach that I deleted one or two PRs ago, is good. https://stackoverflow.com/questions/48956465/favicon-standard-2024-svg-ico-png-and-dimensions
Could you bring back these 3 files from https://github.com/zolcsi/map-of-pi/pull/27/files PR?
Screenshot 2024-02-06 at 09 17 54

Comment (zolcsi) [reply frontend/src/favicon.ico:1]: This icon could be the android-chrome, and the favion.ico should be 16x16, resp. 32x32

**Comment by zolcsi** [comment]
I think there are some even newer icons out there. But I haven't checked their size

**Comment by swoocn** [comment]
Please keep this PR on hold until we get the finalized word from Philip.  Thx!

File: frontend/src/favicon.ico
Comment (swoocn) [reply frontend/src/favicon.ico:1]: Thx for the favicon.io resource!

**Comment by swoocn** [comment]
Philip approves of the logos/ icons.  
Please review and approve the PR accordingly.  Thx!

**Review by zolcsi — APPROVED** [review]
LGTM

**Pull request merged**